cli/commands/preview/lib/validation.ts

```diff
@@ -0,0 +1,24 @@
+import fs from 'fs';
+import path from 'path';
+import { isWordPressDirectory } from 'src/lib/fs-utils';
+import { LoggerError } from 'cli/logger';
+
+function hasWpContentDirectory( projectPath: string ): boolean {
+	return fs.existsSync( path.join( projectPath, 'wp-content' ) );
+}
+
+export function validateSiteFolder( siteFolder: string, action: string ): true {
+	if ( ! fs.existsSync( siteFolder ) ) {
+		throw new LoggerError( `Folder not found: ${ siteFolder }`, action );
+	}
+
+	if ( ! isWordPressDirectory( siteFolder ) && ! hasWpContentDirectory( siteFolder ) ) {
```

Comment:
This is never going to be true, since `isWordPressDirectory` checks if the folder contains a `wp-content` folder.